datahub-web-react/src/app/entity/view/builder/ViewBuilder.tsx

```diff
@@ -23,18 +24,25 @@ type Props = {
  * This component handles creating and editing DataHub Views.
  */
 export const ViewBuilder = ({ mode, urn, initialState, onSubmit, onCancel }: Props) => {
+    const searchVersion = useSearchVersion();
     const userContext = useUserContext();
 
     const client = useApolloClient();
     const [updateViewMutation] = useUpdateViewMutation();
     const [createViewMutation] = useCreateViewMutation();
 
     const emitTrackingEvent = (viewUrn: string, state: ViewBuilderState, isCreate: boolean) => {
+        const filterFields = Array.from(new Set(state.definition?.filter?.filters.map((filter) => filter.field) ?? []));
```

Comment:
what is this ?? thing?

Reply:
haha this was new to me too. `??` is just shorthand for if this exists use it, otherwise default to the other thing.

so here if `state.definition?.filter?.filters.map((filter) => filter.field)` is falsey, use `[]`